tf/environments/dev/main.tf

```diff
@@ -16,7 +16,6 @@ locals {
 ## AWS Setup
 
 provider "aws" {
-  profile = "oonidevops_user"
```

Comment:
This should stay in place as a safety mechanism.

The idea is that you should have your `~/.aws/credentials` setup in such a way that you explicitly define the `oonidevops_user` to point to the access key associated with your personal user account. 

This is especially important when you follow the bootstrap process where you must temporarily create a `oonidevops_root` key associated to the root AWS account of that environment and it's important you don't mix the two up.

You may also have the same key duplicated in the `default` role, but that's fine.

As an example here is how mine is structured:
```
[default]
aws_access_key_id = XXXX
aws_secret_access_key = YYY

[oonidevops_user]
aws_access_key_id = XXXX
aws_secret_access_key = YYY

[other_profile]
aws_access_key_id = ZZZ
aws_secret_access_key = TTT
```

I realize that at the moment you can't do that because your account is not in the authorized accounts list, but I'm going to fix that shortly.

Reply:
Thanks! I did add my IAM account arn to the `Trusted Entities` in the `oonidevops` role temporarily via the UI but I was indeed using the `default` profile. I'll restore the config here and make the necessary changes locally